packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -51,7 +52,7 @@ const Check = styled.span<{ size?: string }>`
         ? `height: 36px;
     width: 36px;`
         : ` height: 16px;
-    width: 16px;`}
+    width: 17.2px;`}
```

Comment:
I think we should use integer pixel values as browsers might round 17.2px differently to either 17px or 18px

Reply:
Okay